pennylane_lightning/lightning_kokkos/lightning_kokkos.toml

```diff
@@ -1,124 +1,109 @@
+DoubleExcitationPlus = { properties = [ "invertible", "differentiable" ] }
+DoubleExcitation = { properties = [ "invertible", "differentiable" ] }
+GlobalPhase = { properties = [ "invertible", "controllable", "differentiable" ] }
+Hadamard = { properties = [ "invertible", "differentiable" ] }
+Identity = { properties = [ "invertible", "differentiable" ] }
+IsingXX = { properties = [ "invertible", "differentiable" ] }
+IsingXY = { properties = [ "invertible", "differentiable" ] }
+IsingYY = { properties = [ "invertible", "differentiable" ] }
+IsingZZ = { properties = [ "invertible", "differentiable" ] }
+MultiRZ = { properties = [ "invertible", "differentiable" ] }
+PauliX = { properties = [ "invertible", "differentiable" ] }
+PauliY = { properties = [ "invertible", "differentiable" ] }
+PauliZ = { properties = [ "invertible", "differentiable" ] }
+PhaseShift = { properties = [ "invertible", "differentiable" ] }
+QubitUnitary = { properties = [ "invertible", "differentiable" ] }
+Rot = { properties = [ "invertible", "differentiable" ] }
+RX = { properties = [ "invertible", "differentiable" ] }
+RY = { properties = [ "invertible", "differentiable" ] }
+RZ = { properties = [ "invertible", "differentiable" ] }
+SingleExcitationMinus = { properties = [ "invertible", "differentiable" ] }
+SingleExcitationPlus = { properties = [ "invertible", "differentiable" ] }
+SingleExcitation = { properties = [ "invertible", "differentiable" ] }
+S = { properties = [ "invertible", "differentiable" ] }
+SWAP = { properties = [ "invertible", "differentiable" ] }
+Toffoli = { properties = [ "invertible", "differentiable" ] }
+T = { properties = [ "invertible", "differentiable" ] }
 
 # Operators that should be decomposed according to the algorithm used
 # by PennyLane's device API.
 # Optional, since gates not listed in this list will typically be decomposed by
 # default, but can be useful to express a deviation from this device's regular
 # strategy in PennyLane.
-decomp = [
-        "BasisState",
-        "QubitStateVector",
-        "StatePrep",
-        "QFT",
-        "MultiControlledX",
-]
+[operators.gates.decomp]
+
+BasisState = {}
+MultiControlledX = {}
+QFT = {}
+QubitStateVector = {}
+StatePrep = {}
 
 # Gates which should be translated to QubitUnitary
-matrix = [
-        "ControlledQubitUnitary",
-        "ECR",
-        "SX",
-        "ISWAP",
-        "PSWAP",
-        "SISWAP",
-        "SQISW",
-        "CPhase",
-        "OrbitalRotation",
-        "QubitCarry",
-        "QubitSum",
-        "DiagonalQubitUnitary",
-        "BlockEncode",
-]
+[operators.gates.matrix]
+
+BlockEncode = {}
+CPhase = {}
+DiagonalQubitUnitary = {}
+ECR = {}
+ISWAP = {}
+OrbitalRotation = {}
+PSWAP = {}
+QubitCarry = {}
+QubitSum = {}
+SISWAP = {}
+SQISW = {}
+SX = {}
+
+# Observables supported by the device
+[operators.observables]
+
+Exp = {}
+Hadamard = {}
+Hamiltonian = {}
+Hermitian = {}
+Identity = {}
+PauliX = {}
+PauliY = {}
+PauliZ = {}
+Prod = {}
```

Comment:
The current consequence of these lines - the mentioned operations will appear in the `device.observables` set. This would prevent PL from decomposing. Again, here we just change the format of the config. The old one listed the same data as list of string [here](https://github.com/PennyLaneAI/pennylane-lightning/blob/2c69c59fa5c2685a461fac6c33c9067f076f59c6/pennylane_lightning/lightning_kokkos/lightning_kokkos.toml#L8).